src/lib/EasyEdit.jsx

```diff
@@ -413,7 +413,7 @@ export default class EasyEdit extends React.Component {
   renderComplexView() {
     const { placeholder, options, type } = this.props;
 
-    if (this.isNullish(this.state.value) || this.state.value.length === 0) {
+    if (this.isNullOrUndefinedOrEmpty(this.state.value) || this.state.value.length === 0) {
```

Comment:
The additional check this.state.value.length === 0 is redundant. The isNullOrUndefinedOrEmpty function already checks if the value is null, undefined, or an empty string, so there is no need to check the length separately.

Reply:
done